cohort-engine/src/main/java/com/ibm/cohort/engine/measure/MeasureEvaluator.java

```diff
@@ -155,7 +156,33 @@ public MeasureReport evaluatePatientMeasure(Measure measure, String patientId, S
 		IMeasureEvaluationSeed seed = seeder.create(measure, periodStart, periodEnd, "ProductLine", parameters);
 
 		CDMMeasureEvaluation evaluation = new CDMMeasureEvaluation(seed.getDataProvider(), seed.getMeasurementPeriod());
-		return evaluation.evaluatePatientMeasure(measure, seed.getContext(), patientId, evidenceOptions, parameters);
+		return evaluation.evaluatePatientMeasure(measure, seed.getContext(), Collections.singletonList(patientId), evidenceOptions, parameters, MeasureReport.MeasureReportType.INDIVIDUAL);
 	}
 
+	public MeasureReport evaluatePatientListMeasure(
+			List<String> patientIds,
+			MeasureContext measureContext,
+			MeasureEvidenceOptions evidenceOptions) {
+		Measure measure = MeasureHelper.loadMeasure(measureContext, measureProvider);
+
+		return evaluatePatientListMeasure(patientIds, measure, measureContext.getParameters(), evidenceOptions);
+	}
+
+	public MeasureReport evaluatePatientListMeasure(
+			List<String> patientIds,
+			Measure measure,
+			Map<String, Parameter> parameters,
+			MeasureEvidenceOptions evidenceOptions) {
+		LibraryLoader libraryLoader = LibraryHelper.createLibraryLoader(libraryProvider);
+
+		MeasureEvaluationSeeder seeder = new MeasureEvaluationSeeder(terminologyProvider, dataProviders, libraryLoader, libraryProvider);
+		seeder.disableDebugLogging();
```

Comment:
In #140, Shelly added the ability to enable debugging for CQL execution. I'd like for that to be consistent across our APIs. We need the ability to enable CQL tracing for debug purposes. It doesn't look like she updated our Measure Evaluation endpoint during her #140 work. I would be ok to spin off a separate ticket for that, but I don't want to lose track of it.

Reply:
Turns out we do have a ticket for enabling CQL debugging - https://jira.wh-sdlc.watson-health.ibm.com/browse/WFFHCOHORT-433